test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -223,6 +234,42 @@ def test_r2plus1d_18_video(self):
         x = torch.randn(1, 3, 4, 112, 112, requires_grad=True)
         self.run_test(model, (x,), rtol=1e-3, atol=1e-5)
 
+    def get_image_from_url(self, url):
+        filename = os.path.basename(urlsplit(url)[2])
+        data_dir = get_writable_path(os.path.join(os.path.dirname(__file__)))
+        path = os.path.join(data_dir, filename)
+        data = request.urlopen(url, timeout=15).read()
+        with open(path, 'wb') as f:
+            f.write(data)
+        image = Image.open(path).convert("RGB")
+        image = image.resize((300, 200), Image.BILINEAR)
+        to_tensor = torchvision.transforms.ToTensor()
+        return to_tensor(image)
+
+    def get_test_images(self):
+        image_url = "https://pytorch.org/tutorials/_static/img/tv_tutorial/tv_image05.png"
+        image = self.get_image_from_url(url=image_url)
+        images = [image]
+        return images
+
+    @skipIfUnsupportedMinOpsetVersion(11)
+    def test_keypoint_rcnn(self):
+        class KeyPointRCNN(torch.nn.Module):
+            def __init__(self):
+                super(KeyPointRCNN, self).__init__()
+                self.model = torchvision.models.detection.keypoint_rcnn.keypointrcnn_resnet50_fpn(pretrained=True,
+                                                                                                  min_size=200,
+                                                                                                  max_size=300)
+
+            def forward(self, images):
+                output = self.model(images)
+                # TODO: The keypoints_scores require the use of Argmax that is updated in ONNX.
+                #       For now we are testing all the output of KeypointRCNN except keypoints_scores.
+                #       Enable When Argmax is updated in ONNX Runtime.
+                return output[0]['boxes'], output[0]['labels'], output[0]['scores'], output[0]['keypoints']
+        images = self.get_test_images()
+        self.run_test(KeyPointRCNN(), (images,), rtol=1e-3, atol=1e-5)
```

Comment:
Can we avoid having PyTorch tests depend on torchvision? This makes for a cyclic dependency on tests, which is not ideal.

Instead, can we add a more minimal test which only exercises the interpolate code-path required by keypoint rcnn, which requires `input_size` to not be a constant?

This will make the test faster, more self-contained and clearer.

Thoughts?

Reply:
Adding this test was requested by Lu and Gregory since it's what the PR claims to add support to (and is the reason we would be including it to 1.4.0).

But regardless, we are planning on adding the tests for fasterrcnn/maskrcnn/keypointrcnn to PyTorch master, since on more than one occasion we have seen test failures on the torchvision CI after some changes were added in PyTorch; this will allow us to identify PRs with breaking changes in PyTorch before they are merged. What do you think?

Reply:
Those are fair points. Let's pin the torchvision version in the PyTorch CI, and update it from time to time, how does this sounds?

Reply:
sounds good!